.github/workflows/test-cluster-set-up.yml

```diff
@@ -0,0 +1,70 @@
+name: Create testing cluster
+
+# Make sure to pass the distribution type and security feature as parameters
+# { 
+#  "distribution": "RPM/DEB/TAR",
+#  "security": "enable/disable"
+# }
+
+on:
+  repository_dispatch:
+    types: [test-cluster-set-up]
+
+jobs:
+  Create-Cluster:
+    name: Create Testing cluster
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_STACK_KEY }}
+          aws-secret-access-key: ${{ secrets.AWS_STACK_SECRET }}
+          aws-region: us-west-2
+
+      - name: Creating cluster
+        run: |
+            #!/bin/bash
+            set -e
+
+            distribution_type=`echo ${{github.event.client_payload.distribution}} | tr [:lower:] [:upper:]`
+            security=`echo ${{github.event.client_payload.security}} | tr [:lower:] [:upper:]`
```

Comment:
Should we have some fallback default values for these variables? In case user forgot to pass it through payload? Or just want to run the workflow without using the dispatch methods.

Reply:
You mean like default ones?

Reply:
Yeah, default values.

Reply:
It will create an unnecessary stack. For eg: if we keep default to RPM with security enable and user (us) wanted to set up deb distribution then we have to delete the already created stack. Better let the script/workflow fail

Reply:
ok sure. Resolved.